manager-bundle/bin/contao-script-handler

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+use Contao\ManagerBundle\Console\ContaoApplication;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Filesystem\Filesystem;
+
+set_time_limit(0);
+@ini_set('zlib.output_compression', '0');
+
+if (file_exists(__DIR__.'/../autoload.php')) {
+    $projectDir = \dirname(__DIR__, 2);
+} elseif (file_exists(__DIR__.'/../../../../autoload.php')) {
+    $projectDir = \dirname(__DIR__, 5);
+} elseif (false !== ($cwd = getcwd()) && file_exists($cwd.'/vendor/autoload.php')) {
+    $projectDir = $cwd;
+} else {
+    $projectDir = \dirname(__DIR__, 4);
+}
+
+require $projectDir.'/vendor/autoload.php';
```

Comment:
keep `$loader = ` here, that has always been the default for Symfony CLI. `global $loader` can be used to access it anywhere in the app (not sure if that's done anywhere, but it can't harm right?)

Reply:
It is a left-over from Symfony 3, which is no longer supported in Symfony 4 and above.